server-base.R

```diff
@@ -224,7 +224,7 @@ shinyServer(function(input, output, session){
   observe({
     region$replot
     input$map_base
-    show_zone_popup <- isolate({input$line_type == 'none'})
```

Comment:
(Nikolai on Robin's PC)
Because it didn't work :cry: it meant was always false.  Have a check, when you select lines (or routes) then the background zone should stop being selectable.